packages/test-utils/src/index.ts

```diff
@@ -0,0 +1,38 @@
+import * as logger from "@changesets/logger";
```

Comment:
Preconstruct/Rollup will fail to build this because reassigning imports is not technically allowed and Rollup errors at compile time on this(but the Babel transform that's running in jest makes it work there), making this a require should be fine.

```suggestion
const logger = require("@changesets/logger");
```